docker-compose/nginx/nginx.conf

```diff
@@ -21,7 +21,7 @@ http {
     }
     # SWE D2CC section
     upstream swe-d2cc-upstream {
-        server swe-d2cc-app;
+        server swe-d2cc-gridcapa-app;
```

Comment:
on peut pas plutôt changer la valeu dans stomp ?

Reply:
to be coherent with the rest of gridcapa apps, the gridcapa common component are identified by the 'gridcapa', so i prefer it this way